ivy_models/transformers/helpers.py

```diff
@@ -22,12 +22,11 @@ def _forward(self, x, **kwargs):
 class FeedForward(ivy.Module):
     def __init__(self, dim, dropout=0., dev_str=None, v=None):
         self._net = ivy.Sequential(
-            ivy.Linear(dim, dim, dev_str=dev_str),
+            ivy.Linear(dim, dim, device=dev_str),
             ivy.GELU(),
-            ivy.Linear(dim, dim, dev_str=dev_str),
-            ivy.Dropout(dropout),
-            dev_str=dev_str)
```

Comment:
was the `device` arg for `ivy.Sequential` removed because it's redundant?

Reply:
I think I had missed it, have added it back